phpstan.dist.neon

```diff
@@ -1,8 +1,14 @@
 parameters:
-    level: 9
+    level: max
     paths:
-        - bin/
-        - config/
-        - public/
         - src/
         - tests/
+    stubFiles:
```

Comment:
My own stubs because Doctrine plugin doesn't work at the moment, at least when used with integrated methods.